cmd/zqd/listen/command.go

```diff
@@ -75,7 +74,7 @@ type Command struct {
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{Command: parent.(*root.Command)}
 	f.StringVar(&c.listenAddr, "l", ":9867", "[addr]:port to listen on")
-	f.StringVar(&c.conf.Root, "datadir", ".", "data directory")
+	f.StringVar(&c.conf.Root, "datadir", "", "data directory")
```

Comment:
```suggestion
	f.StringVar(&c.conf.Root, "datauri", "", "data location")
```
Since this is no longer has to be a directory.

Reply:
`dataloc`?

Reply:
I'd go with `datauri` but feels like the verbiage might confuse users who go the simple route of selecting a location on the file system. Perhaps just the option `data` with @nwt change for the help text "data location". That or `dataloc`.

Reply:
I like -data because it's shorter, but -dataloc also seem fine.